packages/zimic-interceptor/package.json

```diff
@@ -95,7 +95,7 @@
     "@whatwg-node/server": "0.10.10",
     "execa": "9.5.3",
     "isomorphic-ws": "5.0.0",
-    "msw": "2.8.4",
+    "msw": "https://pkg.pr.new/msw@2510",
```

Comment:
We're using the preview release commented [here](https://github.com/mswjs/msw/issues/2146#issuecomment-2922878415) to test the fix. We should migrate to a stable version once it is released.